gui/rendering/Node.hpp

```diff
@@ -75,23 +83,27 @@ class Node : public std::enable_shared_from_this<Node> {
    * Adds another node as a direct child. The node will assume ownership
    * of the node added as a child.
    * @param node
+   * @param localFrameCoord
    */
-  void add(std::shared_ptr<Node> node, bool transformInLocalSpace = true);
+  void add(std::shared_ptr<Node> node, bool localFrameCoord = true);
 
   /**
-   * Removes the node from its parent tree.
+   * Get the root node of the tree
    */
-  void remove();
+  std::shared_ptr<Node> getRoot();
```

Comment:
do we need a shared_ptr here? seems like a non-owning pointer (i.e. weak or raw) would be enough.